calc/src/data/species.ts

```diff
@@ -8237,7 +8237,7 @@ const SS_PATCH: {[name: string]: DeepPartial<SpeciesData>} = {
     types: ['Grass'],
     bs: {hp: 100, at: 125, df: 90, sa: 60, sd: 70, sp: 85},
     weightkg: 80,
```

Comment:
```suggestion
    weightkg: 0,
```

Reply:
Looks like urshifu slipped in as well.  Good catch.